verify/terraform/modules/main.tf

```diff
@@ -104,7 +101,7 @@ resource "azurerm_virtual_machine" "firefoxverify_vm" {
       pass         = "oobeSystem"
       component    = "Microsoft-Windows-Shell-Setup"
       setting_name = "FirstLogonCommands"
-      content      = file("./config/FirstLogonCommands.xml")
+      content      = file("${path.module}/config/FirstLogonCommands.xml")
     }
   }
```

Comment:
Does that work?

It may fail on.

```
module.firefox_verify_env.azurerm_virtual_machine.firefoxverify_vm: Creating...
╷
│ Error: compute.VirtualMachinesClient#CreateOrUpdate: Failure sending request: StatusCode=400 -- Original Error: Code="InvalidParameter" Message="The value of parameter windowsConfiguration.additionalUnattendContent.content is invalid." Target="windowsConfiguration.additionalUnattendContent.content"
│ 
│   with module.firefox_verify_env.azurerm_virtual_machine.firefoxverify_vm,
│   on ../../../modules/main.tf line 54, in resource "azurerm_virtual_machine" "firefoxverify_vm":
│   54: resource "azurerm_virtual_machine" "firefoxverify_vm" {
│ 
╵
```

Reply:
Hmm... In my enviroment that worked, of course.
I will investigate it.

Reply:
Above issue was resolved. It's my  own environment fault.